content/documentation/zooming-images.md

```diff
@@ -0,0 +1,207 @@
+
+With more IIIF images than twelve or so, you’re going to need to host them elsewhere, and then point to them from your project. This is because though small in individual file size, the sheer quantity of files (hundreds and sometimes thousands) that are associated with deep-zooming images can cause issues when trying to run `quire preview`, as well as if you’re using GitHub or a related service to host your project code.
+
+{{< q-class "box tip" >}}
+
+- When quantifying IIIF images, it’s hard to say how many IIIF images will be too many for a system to handle, because the number of individual files associated with a given IIIF image is highly variable.
+
+    **For example:**
+
+    4 levels of zoom ≈ 100 image tiles<br />
+    5 levels of zoom ≈ 329 image tiles<br />
+    6 levels of zoom ≈ 1,170 image tiles
+
+{{< /q-class >}}
+
+For hosting the image tiles, of you have institutional support, your digital department should be able to provide a solution. If you’re on your own, there are any number of options. You might check out [Amazon S3](https://aws.amazon.com/s3/), which is very performant and self-serve but requires some techinical savvy to get set up, or an independent hosting service like [Reclaim Hosting](https://reclaimhosting.com/shared-hosting/), which caters to the academic sector.
+
+## Display IIIF Images in Your Project
+
+To display a IIIF image in your project, you need to point to a IIIF `info.json` file for the image. Either one you’ve identified from an [external source](#use-existing-iiif-images), or one you’ve [created yourself](#creating-your-own-iiif-images) with `quire process --iiif`. If it is an image you processed yourself and it is still in the `processed/` directory, the path to the JSON file would be `/img/iiif/processed/FILENAME/info.json`.
+
+Along with the path to the `info.json` file, you also need to include `media_type: "iiif"` and a path to a lower-res version of the image, hosted in your project’s `static/img` directory (which again is used for Quire’s PDF and e-book outputs). All of this goes in your `figures.yml` file.
+
+```yaml
+- id: "irises"
+  src: "figures/irises.jpg"
+  media_type: "iiif"
+  iiif: "https://data.getty.edu/museum/api/iiif/671108/info.json"
+```
+
+The image can then be added to your Markdown files using the `q-figure-zoom` shortcode. This will display the static image on the page, and then when clicked, will open the figure viewer for the fully zoomable IIIF version.
```

Comment:
Can we give an example of what the shortcode would be `{{< q-figure-zoom id="irises" >}} and maybe connect to the Figure Images section just in case someone skipped that and came straight to IIIF? Maybe we mention that part of the docs in the intro to this section as well?